draft-demarco-oauth-status-attestations.md

```diff
@@ -301,7 +300,8 @@ encoding, for better readability:
     "exp": 1698834139,
     "jti": "6f204f7e-e453-4dfd-814e-9d155319408c",
     "credential_format": "vc+sd-jwt",
```

Comment:
@peppelinux probably we could remove the credential_format. 
WDYT?

Reply:
yes, I'm on it